scrum/tools/sprint_math.py

```diff
@@ -48,6 +48,46 @@ def request_call(url, have_headers):
     return response.json()
 
 
+def delete_card(id):
+    """Makes a DELETE request for the specified card using the requests library.
+
+    Args:
+        id: str, card id (not shortLink)
+
+    Usage:
+        # to delete cards older than <date>
+        from datetime import datetime
```

Comment:
What's with the python in comments? Are these suggestions on how to tweak this to target other boards?

Reply:
It's the loop I ran that called this method